h/models/annotation_metadata.py

```diff
@@ -0,0 +1,26 @@
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql as pg
+from sqlalchemy.ext.mutable import MutableDict
+
+from h.db import Base, types
+
+
+class AnnotationMetadata(Base):
+    __tablename__ = "annotation_metadata"
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    annotation_pk = sa.Column(
+        sa.Integer,
+        sa.ForeignKey("annotation.pk", ondelete="cascade"),
+        nullable=False,
+        unique=True,
+    )
+
+    meta = sa.Column(
```

Comment:
metadata can't be a model attribute. I've gone for meta in the model and metadata in the table.

Should both (model and table) but just be `meta`?

Reply:
How about just `data`?

Reply:
Yeah, I've been using this in some follow up PRs and the meta/metadata idea was awful. `data` it is